draft-ietf-privacypass-key-consistency.md

```diff
@@ -287,105 +299,87 @@ In this model, servers publish keys in an external database and clients fetch ke
 |          |           |
 +----------+           |
                        v
-+----------+         +-+--------+       +----------+
-|          |         |          |       |          |
-|  Client  +-------->+ Database +<------+  Server  |
-|          |         |          |       |          |
-+----------+         +-+--------+       +----------+
-     x                 ^
-     x                 |
++----------+         +-----------+       +----------+
+|          |         |           |       |          |
+|  Client  +-------->+   Cache   +------>+  Server  |
+|          |         |           |       |          |
+|          |         +-----------+       |          |
+|          |                             |          |
+|          |         +-----------+       |          |
+|          |         |           |       |          |
+|          +============ Proxy  ========>+          |
+|          |         |           |       |          |
++----------+         +-+---------+       +----------+
+      x                ^
+      x                |
 +----------+           |
 |          |           |
 |  Client  +-----------+
 |          |
 +----------+
 ~~~
-{: #fig-disc-database title="Database Discovery Example"}
+{: #fig-disc-shared-proxy title="Shared Proxy with Confirmation Discovery Example"}
+
+Ideally, clients confirm with the server via some anonymizing proxy. Examples of proxies
+include anonymous systems such as Tor. Tor proxies are general purpose and operate
+at a lower layer, on arbitrary communication flows, and therefore they are oblivious
+to clients fetching keys. A large set of untrusted proxies that are aware of key fetch
```

Comment:
"large" is an odd word to use here without explanation.

Reply:
Agreed -- dropped the word altogether.